presto-spi/src/main/java/io/prestosql/spi/predicate/TupleDomain.java

```diff
@@ -147,6 +148,36 @@ private TupleDomain(Optional<Map<T, Domain>> domains)
                         })));
     }
 
+    /**
+     * Convert a map of columns to discrete sets of values into the TupleDomain which allows
+     * only those values for the columns. Null is allowed as a value.
+     */
+    public static <T> TupleDomain<T> fromDiscreteSets(Map<T, List<NullableValue>> discreteSets)
+    {
+        return TupleDomain.withColumnDomains(discreteSets.entrySet().stream()
```

Comment:
The method name probably doesn't need to be qualified in the same class

Reply:
After removing tests from `AbstractTestHive `, the method `fromDiscreteSets()` is no longer needed. However, there are multiple other occurrences of qualified calls: `TupleDomain.withColumnDomains()` and `TupleDomain.none()` in the `TupleDomain` class. I thought they were there for clarity. Do you want them cleaned?